test/integration/serving-files.spec.js

```diff
@@ -62,6 +63,30 @@ describe('serves', function() {
       .end(done);
   });
 
+  it('cannot access files above the root', function(done) {
+    var app = connect()
+      .use(superstatic(options()));
+
+    request(app)
+      .get('/../README.md')
+      .expect(404)
+      .end(done);
+  });
+
+  it('file with spaces', function(done) {
+    var opts = options();
+
+    var app = connect()
+      .use(superstatic(opts));
+
+    request(app)
+      .get('/dir/')
```

Comment:
Probably missing something, but how does this test a file with spaces? Isn't it just getting the `index.html`?